### ISSUE TITLE
Orion Convop GET /v1/contextTypes doesn't return types with empty attributes

### ISSUE THREAD

**Issue by anabelengp** [telefonicaid/fiware-orion#686]
We create an entity without any attributes:
```
(curl '10.95.213.244:10026/v1/contextEntities' -H 'Accept: application/json' -H 'Fiware-Service: SmartValencia' -H 'Fiware-ServicePath: /Electricidad' -H 'Content-Type: application/json;charset=UTF-8' -d@- | python -mjson.tool)<<EOF
{
  "id": "test_empty",
  "attributes": [],
  "isPattern": false,
  "type": "empty_attributes"
}
EOF
```
But if we try to GET the contextTypes:
```
(curl 'http://10.95.213.244:10026/v1/contextTypes?attributesFormat=object' -H 'Accept: application/json'  -H 'Fiware-Service: SmartValencia' -H 'Fiware-ServicePath: /Electricidad')
```
The response doesn't include the "empty_attributes" type. (The same happens without the attributeFormat query param)

However, if we create an entity with attributes:
```
(curl '10.95.213.244:10026/v1/contextEntities' -H 'Accept: application/json' -H 'Fiware-Service: SmartValencia' -H 'Fiware-ServicePath: /Electricidad' -H 'Content-Type: application/json;charset=UTF-8' -d@- | python -mjson.tool)<<EOF
{
  "id": "test_empty",
  "attributes": [
    {
      "metadatas": [],
      "name": "aa",
      "type": "aa",
      "value": "aa"
    }
  ],
  "isPattern": false,
  "type": "type_with_attrs"
}
EOF
```
The contextTypes response does include the new type.

Tested against orion > v.0.17.0

**Comment by fgalan** [comment]
Fixed in PR #709, waiting for confirmation from @anabelengp to close it definetively.

**Comment by anabelengp** [comment]
Tested against Orion v0.18.0-next and working OK
